Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,118 @@
+
+h1 {
+	text-align: center;
+	line-height: 0.5;
+	font-size: 20px;
+	font-family: arial;
```

Comment:
I would wrap any font names in single quotes to be safe since they're not globally recognized words of any sort: `font-family: 'Arial', sans-serif;`